_includes/components/svgEmoji.njk

```diff
@@ -0,0 +1,5 @@
+{% macro svgEmoji(emoji = "☕️") %}
+<svg class="emoji" aria-hidden="true" viewBox="0 0 32 32">
+  <text x="0" y="24" dominant-baseline="middle" text-align="middle">☕️</text>
+</svg>
+{% endmacro %}
```

Comment:
Maybe revert the commit that added this because of previous comment on VoiceOver ignoring aria-hidden.